selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/ChangeVariableWithEvaluatingTest.java

```diff
@@ -148,7 +151,15 @@ public void changeVariableTest() throws Exception {
     debugPanel.waitExpectedResultInEvaluateExpression("false");
     debugPanel.clickCloseEvaluateBtn();
     debugPanel.clickOnButton(DebugPanel.DebuggerActionButtons.RESUME_BTN_ID);
-    assertTrue(instToRequestThread.get().contains("Sorry, you failed. Try again later!"));
+    try {
+      assertTrue(instToRequestThread.get().contains("Sorry, you failed. Try again later!"));
+    } catch (AssertionError ex) {
+      LOG.error(
```

Comment:
This log looks redundant here. What is the propose of it?

Reply:
In the log i try to add information from response. This important for clarify what happened when we do not get expected response from test application. So i change error on warn.